gendo.py

```diff
@@ -39,3 +40,37 @@ def quote_of_the_day():
 
 if __name__ == '__main__':
     gendo.run()
+
+
+@gendo.listen_for('swing count')
+def swing_count():
+    today = date.today()
+    now = datetime.now()
+
+    start_of_day = now.replace(hour=9, minute=30)
+    end_date = date(2016, 1, 30)
+    time_remaining = timedelta(days=0)
+
+    d = today
+    while d <= end_date:
+        day_of_week = d.weekday()
+        if day_of_week < 5:
```

Comment:
Ps, indexing time from 0 is confusing. I first read this as if day_of_week < friday